cfdm/read_write/netcdf/netcdfwrite.py

```diff
@@ -2543,12 +2578,13 @@ def _write_netcdf_variable(
         ncvar,
         ncdimensions,
         cfvar,
+        domain_axes,
```

Comment:
I don't see a new docstring parameter entry for this, so we should get one added.

Reply:
Indeed: https://github.com/NCAS-CMS/cfdm/pull/255/commits/7c8d8933fd46aa4003b1879b4a3c2cbb9920b5b7